tailwind.config.js

```diff
@@ -87,7 +88,17 @@ module.exports = {
       tight: 1.2,
       loose: 1.5,
     },
-    opacity: false,
+    opacity: {
+      '0': '0',
+      '20':'0.2',
+      '40':'0.4',
+      '25': '0.25',
+      '50': '0.5',
+      '60': '0.6',
+      '75': '0.75',
+      '80': '0.8',
+      '100': '1',
```

Comment:
plz follow ascending order..... n let's follow gap of 25 for now...we can remove 40, 20, 60, 80